templates/pcw.ini

```diff
@@ -17,26 +17,41 @@ smtp-port = 25
 to = <your_email@suse.com>
 from = pcw@publiccloud.qa.suse.de
 
+
+[cluster.notify]
+# List of people who will be notfied about found EKS clusters
+to = <your_email@suse.com>
+
 [notify.namespace.qac]
 # list of recipients which will be notified in case of issues in certain namespace
 to = <your_email@suse.com>
 
 [cleanup]
-# Specify how many images per flavor get kept
-max-images-per-flavor = 2
-# Max age of an image file
-max-images-age-hours = 24
+# Max age of data storage resources ( used in EC2 only )
+ec2-max-age-days = 1
+# Max age of data storage resources ( used in Azure and GCE )
+max-age-hours = 1
 # Specify with which namespace, we will do the cleanup.
 # if not specifed default/namespaces list will be taken instead
 namespaces = qac
 
 # Values specified under "cleanup.namespace.{namespace}" have precedence over same values in [cleanup] for this certain namespace
 [cleanup.namespace.qac]
-# EC2 snapshots younger than this amount of days will be ignored
-ec2-max-snapshot-age-days = 2
+# Max age of an image file ( used in EC2 only )
+ec2-max-age-days = 2
 # EC2 volumes younger than this amount of days will be ignored
 ec2-max-volumes-age-days = 2
```

Comment:
I think this value isn't there as well or?
```suggestion

```

Reply:
yes , you right , removing